src/preferences/dialog/dlgprefmixer.cpp

```diff
@@ -57,6 +71,26 @@ DlgPrefEQ::DlgPrefEQ(
           m_bEqAutoReset(false),
           m_bGainAutoReset(false) {
     setupUi(this);
+
+    QButtonGroup crossfaderModes;
+    crossfaderModes.addButton(radioButtonAdditive);
+    crossfaderModes.addButton(radioButtonConstantPower);
+
+    loadSettings();
+
+    connect(SliderXFader,
+            QOverload<int>::of(&QSlider::valueChanged),
+            this,
+            &DlgPrefMixer::slotUpdateXFader);
+    connect(SliderXFader, &QSlider::sliderMoved, this, &DlgPrefMixer::slotUpdateXFader);
+    connect(SliderXFader, &QSlider::sliderReleased, this, &DlgPrefMixer::slotUpdateXFader);
+    connect(SliderXFader, &QSlider::sliderReleased, this, &DlgPrefMixer::slotApply);
```

Comment:
I doubt its a good UX if simply releasing the slider instantly applies the settings.

Reply:
even more so if release applies **all** pending changes.

Reply:
Yes.